packages/terra-demographics-banner/src/index.js

```diff
@@ -0,0 +1,5 @@
+import DemographicsBanner from './DemographicsBanner';
+import SelectableDemographicsBanner from './selectable-demographics-banner/SelectableDemographicsBanner';
+
+export default DemographicsBanner;
+export { SelectableDemographicsBanner };
```

Comment:
do we want to offer imports from two places? This will load assets for both with webpack, making the bundle larger. I agree it's convenient, but would increased the bundle size.

Reply:
Initial Discussion was to  provide selectable-demographics banner as a separate import. hence added it here. would like to know if we can do this in better way.

Reply:
Components can be imported directly from lib to keep the bundle sizes smaller. The index.js file and main entry change in the package.json should be unnecessary.

```jsx
import SelectableDemographicsBanner from 'terra-demographics-banner/lib/SelectableDemographicsBanner';
```

Check out the specialized field components(InputField, TextareaField, etc..) for examples:

https://engineering.cerner.com/terra-ui/components/terra-form-input/form-input/input-field

Reply:
removed index.js and updated imports [here](https://github.com/cerner/terra-core/pull/2856/files#diff-80d0f193afc74e71713ee540cd315332)